minecraft/src/main/java/org/quiltmc/loader/impl/game/minecraft/MinecraftGameProvider.java

```diff
@@ -474,6 +474,8 @@ public void launch(ClassLoader loader) {
 			targetClass = "org.quiltmc.loader.impl.game.minecraft.applet.AppletMain";
```

Comment:
```suggestion
			GameTransformer.appletMainClass = targetClass;
			targetClass = "org.quiltmc.loader.impl.game.minecraft.applet.AppletMain";
```

This could've easily satisfy both AppletLauncher classes instead of adding a dependency on `LibClassifier` and adding more code at the end of it.